admin/c2cgeoportal_admin/templates/navigation.jinja2

```diff
@@ -0,0 +1,10 @@
+<ul class="nav nav-pills nav-stacked">
+  <li role="presentation" class="{{'active' if request.matched_route.name == 'layertree' else ''}}">
```

Comment:
Rooooh, according to mockups, need two levels in menu bar !

Reply:
Yes, I've seen that, but let's do it in another pull request.